### PR TITLE
Add distance param to documentation

### PR DESCRIPTION
**Pull request by vinzentt**

### CODE REVIEW

File: documentation/navitia.io/source/integration.rst [hunk 1]
```diff
@@ -273,7 +273,7 @@ Once you have selected a region, you can explore the public transportation objec
 easily with these apis. You just need to add at the end of your url 
 a collection name to see all the objects of a particular collection.
 To see an object add the id of this object at the end of the collection's url.
-The only arguments are the ones of `paging`_.
+The arguments are the ones of `paging`_.
```
Comment (stifoon) [inline documentation/navitia.io/source/integration.rst]: I would have written "You can add arguments from `paging`_."

Comment (vinzentt) [inline documentation/navitia.io/source/integration.rst]: You mean you would not have written what you wrote? =)
Personally, I would prefer sthg like "`paging`_ arguments may be used to paginate results."

Comment (stifoon) [inline documentation/navitia.io/source/integration.rst]: na na na na na...
Yes, your proposition is fine. Nay better than mine :)

**Comment by antoine-de** [comment]
thanks for the PR @vinzentt !

**Pull request merged**